docs/design/debug_vllm_compile.md

```diff
@@ -0,0 +1,218 @@
+# How to debug the vLLM-torch.compile integration
+
+TL;DR:
+
+- use tlparse to acquire torch.compile logs. Include these logs in bug reports and/or support asks.
+- The vLLM-torch.compile integration is multiple pieces. vLLM exposes flags to turn off each piece.
```

Comment:
I wonder if it would be useful just to add a table of all the typical configs like
| Online Flag | Offline Flag   |      Result |
|----------|----------|-------------|
| --enforce-eager | enforce_eager=True |  Turn off vLLM-torch.compile |
| -O.mode=0 | mode=CompilationMode.NONE |  Turn off torch.compile |
| -O.cudagraph_mode=NONE | compilation_config=CompilationConfig(mode=CompilationMode.NONE) |  Turn off CUDAGraphs |
| -O.use_inductor=False | compilation_config=CompilationConfig(use_inductor=False) |  Turn off TorchInductor |